frontend/src/components/Login.jsx

```diff
@@ -13,19 +13,29 @@ class Login extends Component {
   }
 
   facebookLogin(response) {
-    console.log("fb response", response);
     const { fbLogin } = this.props;
-    const { first_name, last_name, accessToken, email, id, picture } = response;
-    const user = {
-      firstName: first_name,
-      lastName: last_name,
-      email,
-      facebookId: id,
-      imageUrl: picture.data.url
-    };
-
-    const auth = { token: accessToken, user };
-    fbLogin(auth);
+    const { first_name, last_name, accessToken, email, id, picture, accounts} = response;
+    console.log('ig_business', accounts.data)
```

Comment:
Accessing `accounts.data` results to an error, because `accounts` is undefined. Are you encountering this error too @effyyzhang ?